configs/rbac_cel_custom_library.yaml

```diff
@@ -0,0 +1,197 @@
+# RBAC with policies based on CEL expressions with custom library
+# curl -v "localhost:7777" will proxy to localhost:9901, the admin page
+
+admin:
+  address:
+    socket_address:
+      protocol: TCP
+      address: 127.0.0.1
+      port_value: 9901
+static_resources:
+  listeners:
+  - name: localhost-listener
+    address:
+      socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 7777
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          access_log:
+          - name: envoy.access_loggers.stdout
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StdoutAccessLog
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match:
+                  prefix: "/"
+                route:
+                  cluster: admin_cluster
+          http_filters:
+          - name: rbac-deny-filter-using-custom-library
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC
+              rules:
+                action: DENY
+                custom_library_config:
+                  name: envoy.rbac.custom_library_config.custom_library
+                  typed_config:
+                    "@type": type.googleapis.com/envoy.extensions.rbac.custom_library_config.v3.CustomLibraryConfig
```

Comment:
Is this an example CEL extension? I'm wondering what its intent is given it has a field governing its replacement semantics but not anything else.